examples/typescript/components/http-hello-world/package.json

```diff
@@ -4,10 +4,11 @@
   "description": "A WebAssembly Component that responds to HTTP requests using the `wasi:http` WIT interface",
   "main": "dist/http-hello-world.js",
   "scripts": {
+    "generate:types": "jco types wit/ -o generated/types",
     "build:tsc": "tsc",
     "build:js": "jco componentize -w wit -o dist/http-hello-world.wasm dist/http-hello-world.js",
     "build:component": "wash build --sign-only --config-path wasmcloud.toml",
-    "build": "npm run build:tsc && npm run build:js && npm run build:component",
+    "build": "npm run generate:types && npm run build:tsc && npm run build:js && npm run build:component",
```

Comment:
Hey, just a question here. I am not familiar with jco and its internals, but would it make sense to clean the folder before putting the generated files there? I mean if it is changing in the future are the generated folder recreated? 🤔 
Maybe `generate:types` might have a nice `rimraf` or similar. I know that this is a template and the chance is low to litter the folder here, but the peeps who are using it might do so.

Reply:
Oh yeah this is a great point -- here's [an issue](https://github.com/wasmCloud/wasmCloud/issues/3856) to track this

IIRC we're going to move some examples out of the main repo to make them a bit easier to maintain independently, so this change could be made once that's done!